CONTRIBUTING.md

```diff
@@ -13,11 +13,36 @@ Note that it's important that `isort` is run first, because it doesn't produce a
 
 If you push changes to main or create a pull request, please be aware that Github Actions will trigger a workflow that runs `isort` and `black` on the code. This will take a few seconds to run and the workflow may automatically push formatting changes to the repository. To ensure your local repository is up to date with the remote repository, wait for a few seconds and pull the latest changes.
 
-### Branch naming
+# Branch naming
 If you are adding a branch to this repository, please use the following convention: `{feature, bugfix, hotfix, release, docs}/{developer initials}/{short-hyphenated-description}`. For example, `docs/DR/add-branch-naming-convention` for this change. For a description of the prefixes, please see [here](https://medium.com/@abhay.pixolo/naming-conventions-for-git-branches-a-cheatsheet-8549feca2534).
 
-### Docstrings
+# Docstrings
 For documentation, we use the [Google](https://github.com/NilsJPWerner/autoDocstring/blob/HEAD/docs/google.md) format. I personally use [VSCode autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) plugin for templating. 
 
-### Type hints
+# Type hints
 Typing hints, as introduced by [PEP 484](https://peps.python.org/pep-0484/), are strongly encouraged. This helps provide additional documentation and allows some code editors to make additional autocompletes. 
+
+# Contributing to MkDocs
```

Comment:
A little more of a summary here would be helpful. And it would be good to explain it at a slightly higher level. Rather than saying something specific about the tool "MkDocs" I'd say something more general about what they are doing. Like "Updating documentation"